stdlib/public/runtime/ErrorObject.mm

```diff
@@ -153,16 +167,29 @@ - (BOOL)isEqual:(id)other {
 @end
 
 Class swift::getNSErrorClass() {
-  return SWIFT_LAZY_CONSTANT([NSError class]);
+  return SWIFT_LAZY_CONSTANT(objc_lookUpClass("NSError"));
 }
 
 const Metadata *swift::getNSErrorMetadata() {
   return SWIFT_LAZY_CONSTANT(
     swift_getObjCClassMetadata((const ClassMetadata *)getNSErrorClass()));
 }
 
+static Class getAndBridgeSwiftNativeNSErrorClass() {
+  Class nsErrorClass = swift::getNSErrorClass();
+  Class ourClass = objc_lookUpClass("__SwiftNativeNSError");
```

Comment:
This one you should be able to refer to statically, no? It's right in this file.

Reply:
yeah I suppose I can do [__SwiftNativeNSError class]. sadly I can't just do __SwiftNativeNSError or @class(__SwiftNativeNSError) or something.